### PR TITLE
Adding support for autogenerating dots

### LINKED ISSUE #28: Move dots
Place dots where they belong on the map. This may involve messing with the tilemap.

Top left dot position: `[25, 25]`
Bottom right dot position: `[425, 475]`

### PR DESCRIPTION
**Pull request by joshbarbee**
Automatically generates dots at all non-spawn locations.

Potential Issues:
- collides property - we can't test whether the .collides property works as intended on tiles, since the black tiles we used to block spawning dots in the ghost base/pipes may/may not block movement. 

Closes #28

### CODE REVIEW

File: assets/map.json [hunk 1]
```diff
@@ -399,6 +404,10 @@
                         }]
                 }],
          "tilewidth":16
+        }, 
+        {
+         "firstgid":49,
+         "source":"..\/..\/..\/pacman-tilemap\/Pacman.tsx"
```
Comment (adambricelis) [inline assets/map.json:410]: This doesn't look right!

**Review by adambricelis — CHANGES_REQUESTED** [review]

File: assets/map.json (hunk 1 above)
Comment (joshbarbee) [reply assets/map.json:410]: fixed in commit 572e78a

File: script.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ var worldLayer;
 
 var game = new Phaser.Game(config);
 
+const tileSize = 16
```
Comment (adambricelis) [inline script.js:39]: Could you move this up with the other constants and make it ALL_CAPS?

**Review by adambricelis — APPROVED** [review]
Nice work!

**Pull request merged**